File: leetcode/976/976.py
class Solution:
    def largestPerimeter(self, A: List[int]) -> int:
        A.sort(reverse=True)
        l = len(A)
        for i in range(2, l):
            if A[i-2] < A[i-1] + A[i]:
                return A[i-2] + A[i-1] + A[i]
        return 0

File: leetcode/976/976.go
package main

import (
	"sort"
)

func largestPerimeter(A []int) int {
	sort.Sort(sort.Reverse(sort.IntSlice(A)))
	for i := 2; i < len(A); i++ {
		if A[i-2] < A[i-1]+A[i] {
			return A[i-2] + A[i-1] + A[i]
		}
	}
	return 0
}
